### ISSUE TITLE
SIP relay board for raspberry pi 3

### ISSUE THREAD

**Issue by Riaan111209** [KanyonKris/relay_board#5]
Good day.
Do I use the same pins for the raspberry pi 3-40pins

**Comment by DarthKegRaider** [comment]
Yes.  I'm using a Pi Zero W, which uses the same GPIO as a Pi3, and I have no issues whatsoever.  I know this is many years after the "Issue", but if someone else is reading then yes, it works fine with later model Raspberry Pi's.
